Applications/Custom/mae_loss.h

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file   mae_loss.h
+ * @date   10 June 2021
+ * @brief  This file contains the mean absoulte error loss as a sample layer
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug    No known bugs except for NYI items
+ *
+ */
+#ifndef __MAE_LOSS_LAYER_H__
+#define __MAE_LOSS_LAYER_H__
+#include <string>
+
+/// @todo migrate these to API and ensure those headers are exposed to devel
+#include <layer_internal.h>
+#include <manager.h>
+#include <tensor.h>
+
+namespace custom {
+
+/**
+ * @brief A sample loss layer which calculates mean absolute error from output
+ *
```

Comment:
add a todo to update inheritance from layer_devel?

Reply:
Done